### ISSUE TITLE
Release numbering tie to Polymer version?

### ISSUE THREAD

**Issue by kaceo** [PolymerElements/polymer-starter-kit#1078]
I notice the current releases of Polymer Starter Kit is 3.2 but not upgraded to Polymer 3.0 architecture.  Is there plan to tie the major release numbering so that there is less confusion?

**Comment by TimvdLippe** [comment]
Hm I am not sure. @abdonrd @robdodson any thoughts on this? If we want to comply with semver, Polymer 3 will have a PSK 4 version.

**Comment by abdonrd** [comment]
My thought is to comply the semver, not the Polymer versions. So...

- `PSK ^1.0.0` is built in Polymer v1 with gulp tasks.
- `PSK ^2.0.0` is built in Polymer v1 with Polymer CLI.
- `PSK ^3.0.0` is built in Polymer v2 (keeping Polymer CLI).
- `PSK ^4.0.0` is very likely to be built in Polymer v3, since there is nothing special thought for that version..

**Comment by robdodson** [comment]
Yeah I agree with abdon. We kinda ended up in a weird spot with semver but
I don't think we can go back now 😝
